mmdet/models/detectors/base.py

```diff
@@ -304,13 +304,40 @@ def show_result(self,
             bbox_result, segm_result = result
             if isinstance(segm_result, tuple):
                 segm_result = segm_result[0]  # ms rcnn
+            if bbox_result[0].shape[1] == 5:  # bbox-segm two stage
```

Comment:
We can allow to not draw bounding boxes if we only have segm and do not need to manually draw bounding boxes for that.

Reply:
The description of two stage is wrong, because yolact is not a two-stage.

Reply:
In extreme cases, it is best to add a unit test.